Client/mods/deathmatch/logic/CClientPed.cpp

```diff
@@ -2377,6 +2378,9 @@ eMovementState CClientPed::GetMovementState()
         else if (strcmp(szComplexTaskName, "TASK_COMPLEX_JUMP") == 0)
             return MOVEMENTSTATE_JUMP;
 
+        else if (IsEnteringVehicle())
```

Comment:
```suggestion
        else if (IsEnteringVehicle() && strcmp(szSimpleTaskName, "TASK_SIMPLE_GO_TO_POINT") != 0)
```
Did a lot of tests and the `TASK_SIMPLE_GO_TO_POINT` task in MTA is only performed when the player walks to a vehicle to enter it and this is what is needed to solve #1594.